meilisearch/index.py

```diff
@@ -2057,10 +2058,15 @@ def get_localized_attributes(self) -> Union[List[Dict[str, List[str]]], None]:
         MeilisearchApiError
             An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
         """
-        return self.http.get(self.__settings_url_for(self.config.paths.localized_attributes))
+        response = self.http.get(self.__settings_url_for(self.config.paths.localized_attributes))
+
+        if not response:
+            return None
+
+        return [LocalizedAttributes(**attrs) for attrs in response]
```

Comment:
Saw this pattern elsewhere in the file, although I think it could also be:
```suggestion
        if response:
            return [LocalizedAttributes(**attrs) for attrs in response]

```

Reply:
The way you have it is good. Technically your other idea would work also and do the same thing, but the type checker will most likely complain because it won't know if you are ommiting the `return None` on purpose of if it was overlooked.